linera-core/src/unit_tests/test_utils.rs

```diff
@@ -1136,11 +1136,11 @@ impl StorageBuilder for ScyllaDbStorageBuilder {
     }
 }
 
-/// Creates `count` [`MockApplication`]s and registers them in the provided [`ExecutionStateView`].
+/// Creates `count` [`MockApplication`]s and adds the blobs to storage.
 ///
 /// Returns an iterator over a tuple of [`UserApplicationId`]s and their respective
 /// [`UserApplicationDescription`]s, [`MockApplication`]s, and contract and service [`Blob`]s.
-pub async fn register_mock_applications<C, S>(
+pub async fn create_mock_applications<C, S>(
```

Comment:
Nit: I think "create" here provides incomplete semantics of what it's doing. Maybe we should name it `upload_mock_applications`, or some other verb that puts emphasis on the adding to blob storage part ("initialize", "setup"?). A fallback could be to keep the old name :thinking: